clients/web/test/testUtils.js

```diff
@@ -11,6 +11,17 @@ window.alert = function (msg) {
     expect('Alert was used').toBe('Alerts should not be present');
 };
 
+// Augment jQuery's click() function to follow href properties in links if they exist.
+$.fn.click = _.wrap($.fn.click, function (old) {
+    var args = Array.prototype.slice.call(arguments, 1);
+    var href = $(this).attr('href');
+    if (args.length === 0 && href) {
+        window.location = href;
```

Comment:
I'm having trouble consistently reproducing the `click` failure on an actual Girder site. Is it possible that this is a browser-based issue (i.e. PhantomJS)? Can you provide more details in code comments on when this workaround is necessary, and when it could be removed in the future?

Reply:
Are you actually clicking, or calling jquery `click()` to try and reproduce it?

Reply:
I was slightly mistaken initially, I was calling jQuery's `.click` on something that coincidentally also had Backbone behavior on it. Now I'm able to reproduce the bug.

However, per https://stackoverflow.com/a/21762745 I am able to make things work properly by `.click()`ing the underlying DOM element. I think it'd be safer for this method to just unwrap the jQuery selector and call [`click()` on the native element](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click). That comes back around to my other question: what is the type of `this` here?